Makefile

```diff
@@ -12,7 +12,7 @@ all: build test check-coverage lint lint-protos ## build, test, check coverage a
 
 ci: clean check-uptodate all  ## Full clean build and up-to-date checks as run on CI
 
-check-uptodate: protos
+check-uptodate: protos pb
```

Comment:
i did initially rename it to `echo.EchoService.pb`, but named it back when I noticed the `pb` target in the `Makefile` built all the protos it could find. When I implement multiple method/service directories, I will address this.